peekingduck/configs/output/screen.yml

```diff
@@ -1,2 +1,8 @@
 input: ["img"]
-output: ["pipeline_end"]
\ No newline at end of file
+output: ["pipeline_end"]
+
+window_name: PeekingDuck
+window_width: 1280
+window_height: 720
+window_x_coord: 0
+window_y_coord: 0
```

Comment:
Suggest OOP-ing the above with a `window` object:
```
window: {
    name: "PeekingDuck",
    x: 0,
    y: 0,
    width: 1280,
    height: 720,
}
```
Facilitates future attribute extension.

Reply:
I was thinking of this at first, but thought it would be clunky when users read our RTD page. I'd have to lump all the descriptions under one `Dict` type which can look messy with different descriptions. Also there's a mix of `int` and `str` types that get lost when all are lumped under `Dict`.

Example from `resize` config from `input.live` - I think this is simple enough to be nested though.
![image](https://user-images.githubusercontent.com/33995028/144352321-d178989f-9d9f-4276-8733-2bb4f030d419.png)

What do you think?

Reply:
Alternatively, I could do it this way, which is a good compromise:
```
window_name: "PeekingDuck"
window_loc: {
   x: 0,
   y: 0 }
window_size: {
   width: 1280,
   height: 720 }
```

Reply:
1. The `resize (Dict)` example also relies on the user to infer the `bool` and `int` types, so it actually servers to highlight the point made on lost type info.
2. The alternative you suggested is a compromise that I am neutral about.
3. Another suggestion is to remove `window_` totally, just have
```
inputs = 
outputs =

name: "PeekingDuck"
x: 0
y: 0
width: 1280
height: 720
```
and the code will be `self.height`, `self.width`, etc.
Since, after all, the `screen` is the display.

Reply:
Tried the last suggestion, but the keyword `name` causes an `AttributeError: can't set attribute`.

In the end I went for the in-between solution, the RTD looks like this:
![image](https://user-images.githubusercontent.com/33995028/144385444-f20365a7-d954-49e1-a1bb-ec8912c5e40f.png)

Reply:
Cool, let's go with the in-between solution.
Good reminder that `name` is reserved, for future similar cases, we'll have to use `title` or `label`.

Reply:
`resizeWindow()` doesn't work for opencv >= 4.5.x, only applying `resize()` to the image works. Unfortunately, it is the opposite case for opencv == 4.1.2.30. To avoid discrepancies in behaviour, in this PR, will go with the solution that works for 4.5.x, and will create another PR to upgrade our openCV version (issue #556).

Added a boolean config for resizing window that's consistent with `input.live` and `input.recorded`.